### ISSUE TITLE
Socker Erro catch

### ISSUE THREAD

**Issue by cubecleveland** [AxisCommunications/media-stream-library-js#669]
Hello,

We are using the HTTP over RTSP feature in the axis camera.

Currently the app breaks when the server is down or closing the connection,

React.useEffect(() => {
    //  if (!data) return;

    Pipeline = pipelines.Html5VideoPipeline;
    mediaElement = ref.current;
    // hide the other output

    // Setup a new pipeline
    pipeline = new Pipeline({
      ws: {
        uri: `ws://localhost:9200`,
      },
      rtsp: {
        // uri: `rtsp://${camera.ip}:554/axis-media/media.amp?videocodec=h264&audio=1`,
      },
      mediaElement,
    });

    pipeline.rtsp.onRtcp = (rtcp) => {
      if (isRtcpBye(rtcp)) {
        setTimeout(() => pipeline.rtsp.play(), 0);
      }
    };

    // Restart stream on RTCP BYE (stream ended)

    pipeline.ready.then(() => {
      pipeline.rtsp.play();
    
    });

    return () => {
      pipeline && pipeline.rtsp.close();
    };
  }, []);

basically how can we catch all eorrs if the socket url is wrong and can not connect?
We get this message in the console when socket is not avilable:

 Uncaught DOMException: Failed to construct 'WebSocket': The URL contains a fragment identifier

**Comment by cubecleveland** [comment]
fixed. there was an issue on the server side terminating the socket.
